addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -291,6 +291,13 @@ $o-wblog-loader-size: 50px;
             .o_wblog_post_cover_nocard .o_record_cover_container {
                 padding-top: 33%;
             }
+
+            @include media-breakpoint-down(md) {
+                > .row {
+                    margin-left: 0 !important;
+                    margin-right: 0 !important;
```

Comment:
Why is the !important needed? I tried removing it and it seemed to go fine. 🤔  I think not having it would be better since this is a fix in a stable version, and we don't want to override too much styles that the client might have made.

Reply:
Indeed, it is not necessary :open_mouth:  I thought there was `!important` on the row margin rules but I in fact imagined it haha 
I will remove them, as the selector is more specific anyway :slightly_smiling_face: